chunky/src/java/se/llbit/chunky/block/Chain.java

```diff
@@ -6,13 +6,28 @@
 import se.llbit.math.Ray;
 
 public class Chain extends MinecraftBlockTranslucent {
-  public Chain(String name, Texture texture) {
+
+  private final int axis;
```

Comment:
Instead of having this magic number here, let's keep it a string here and move the conversion to an array index into the `ChainModel`

Reply:
Ok, sounds good. I adapted this from the End Rod classes, so should this be changed there too?

Reply:
Yeah... Legacy code from back when `data` actually _was_ a number (pre-flattening, i.e. 1.12). No need to go through all that code but I tried to use the new pattern for new blocks when adding 1.14-1.16 support.